integrations/node/install-driver.sh

```diff
@@ -47,3 +47,7 @@ npm install yargs lodash.omit
 cd ${PROJECT_DIRECTORY}
 npm install
 
+# compile typescript if necessary
```

Comment:
this is the change to the driver we discussed yesterday that I think would obviate this step:
```
diff --git a/package.json b/package.json
index 32df1e850..a5896ab70 100644
--- a/package.json
+++ b/package.json
@@ -81,7 +81,7 @@
     "check:ts": "tsc --noEmit",
     "check:atlas": "mocha --config \"test/manual/mocharc.json\" test/manual/atlas_connectivity.test.js",
     "check:ocsp": "mocha --config \"test/manual/mocharc.json\" test/manual/ocsp_support.test.js",
-    "prepublishOnly": "npm run build:ts",
+    "prepare": "npm run build:ts",
     "release": "standard-version -i HISTORY.md",
     "test": "npm run check:lint && npm run check:test"
   },
```

Reply:
I think the prepare script will have us covered, the npm install above will run the prepare script so the driver will be compiled if it's from master.

Reply:
Updated, PTAL